test/run_all.py

```diff
@@ -15,10 +15,6 @@
     m = __import__(os.path.splitext(f)[0])
     testsuite.addTests(unittest.TestLoader().loadTestsFromModule(m))
 
-# Run the test suite.
-def test(verbosity=1):
-    runner = unittest.TextTestRunner(verbosity=verbosity)
-    runner.run(testsuite)
```

Comment:
This is unused. I can revert it if you want b/c it is kind of unrelated to this PR.